ingestion/src/airflow_provider_openmetadata/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenMetadata Airflow Provider
+
+This package brings:
+- Lineage Backend
+- Lineage Operator
+- OpenMetadata Hook
+
+Note that this is configured as an entrypoint in the `setup.py`:
+
+```python
+entry_points={
+    "apache_airflow_provider": [
+        "provider_info = airflow_provider_openmetadata:get_provider_config"
+    ],
+},
+```
+
+Therefore, any metadata changes that should be discoverable by Airflow need to be passed in `get_provider_config`.
+
+More information about that on Airflow's [docs](https://airflow.apache.org/docs/apache-airflow-providers/index.html?utm_cta=website-events-featured-summit#creating-your-own-providers).
+
+## How to use the OpenMetadataHook
+
+In the Airflow UI you can create a new OpenMetadata connection.
```

Comment:
can we showcase some example with screenshots on how to create the connection. Especially we have nested fields

Reply:
Added that in the lineage operator docs